.knip.json

```diff
@@ -0,0 +1,26 @@
+{
+  "ignore": [
+    "lighthouserc.js",
+    ".lighthouseci/**",
+    ".github/**",
+    "microsite/**",
+    "contrib/**",
+    "scripts/**",
+    "**/*.stories.*",
+    "**/.eslintrc.js",
+    "**/setupTests.ts",
```

Comment:
Hm. What does "ignore" mean in this context? `setupTests`, `/dev/` files etc SHOULD be taken into account when deciding whether a dependency is unused or not. For example the `setupTests.ts` file is typically the only place where `@testing-library/jest-dom` is referenced and that means it should stay as a dev dependency for those packages.

But they should NOT be taken into account in the check for unused files.

Interestingly, the `scripts` folder is the cause of a lot of the dependencies in the root `package.json`, which need to stay around.